grunt/aliases.yaml

```diff
@@ -1,38 +1,16 @@
-# Where we tell Grunt what to do when we type "grunt" into the terminal
-# $ grunt
 default:
-  - 'clean'
-  - 'sass:dist'
-  - 'assemble'
-  - 'juice'
-  - 'imagemin'
-  - 'replace:src_images'
+  - serve
 
-# Use grunt send if you want to actually send the email to your inbox
-# $ grunt send --template=transaction.html
-send:
-  - 'mailgun'
+build:
+  - clean
+  - sass:dist
+  - assemble
+  - juice
 
-# Upload images to your CDN on Rackspace Cloud Files
-# $ grunt rsupload
-rsupload:
-  - 'default'
-  - 'cloudfiles'
-  - 'cdn:cloudfiles'
-
-# Upload image files to Amazon S3
```

Comment:
Similarly, I think the S3 task is important and convenient, and since it is not part of the default task is fine from a GDPR perspective to retain.

Reply:
Done. Testing required.